snapcraft.yaml

```diff
@@ -55,6 +55,8 @@ parts:
     build-packages:
       - shellcheck
       - distro-info
+    build-snaps:
```

Comment:
Obviously would have had bootstrap issues. For core26 we will have a slightly different solution anyway due to the fact of it being chiselled.